Duplicati/Library/Main/default_compressed_extensions.txt

```diff
@@ -93,11 +93,14 @@
 .jpg #JPEG Image
 .pamp #PowerAmp Image Cache
 .pdn #Paint.NET Image
+.pld # Photoline Image
+.plb # Photoline Image Cache
```

Comment:
It appears that `pld` and `plb` are also used by other applications (Oracle, Adobe Premiere, ...).  I don't know how common these are, but if these extensions aren't primarily associated with Photoline, we might not want to include them here by default if the Oracle and Adobe usages can benefit from compression.

Note that the user can also provide a custom extensions file using the `--compression-extension-file` option if necessary.